tests/unit/moduleapi/usercall.tcl

```diff
@@ -90,5 +124,13 @@ start_server {tags {"modules usercall"}} {
         assert_match {*ERR ACL failure in script*} $e
 
         assert_equal [r usercall.call_with_user_flag C evalsha $sha_get 0] 1
+
+        # verify that new log entry added
+        set entry [lindex [r ACL LOG] 0]
+        assert_equal [dict get $entry username] {module_user}
+        assert_equal [dict get $entry context] {lua}
+        assert_equal [dict get $entry object] {set}
+        assert_equal [dict get $entry reason] {command}
+        assert_match {*cmd=usercall.call_with_user_flag*} [dict get $entry client-info]
```

Comment:
before the fix it used to return "cmd=evalsha" instead of "cmd=usercall.call_with_user_flag"